doc/release/1.9.0-notes.rst

```diff
@@ -26,42 +26,117 @@ For running on PyPy, PyPy3 6.0+ is required.
 Highlights of this release
 **************************
 
+- Added `scipy.optimize.milp`, new function for mixed-integer linear
+  programming.
+- Added `scipy.stats.fit` for fitting discrete and continuous distributions
+  to data.
+- Added new spline based interpolation methods for
+  `scipy.interpolate.RegularGridInterpolator`.
```

Comment:
Maybe rephrase to read "Tensor-product spline interpolation modes were added to `scipy.interpolate.RegularGridInterpolator`" or some such.